MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController.swift

```diff
@@ -36,6 +36,20 @@ class MovieDetailViewController: UIViewController {
         
         presenter.fetchAll(for: movieId)
     }
+
+    override func viewDidLayoutSubviews() {
+        var height: CGFloat = 0
+        contentView.subviews.forEach {
+            height += $0.frame.height
+        }
+        
+        height += 34 * offset + 30
+        
+        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: height)
+        contentView.snp.updateConstraints {
+            $0.height.equalTo(height)
+        }
```

Comment:
Scroll view can calculate this automatically if every subview has fixed height.

Reply:
[Here](https://github.com/lihter/Movie-App/pull/35/commits/bea08e4c4026e5a13e5d0634d3a88cd505dff0d6#diff-f5b01144bf2cdd8175add69b29bdfdcf3abf149a38fdd4929f07da30b107f914L37)